### PR TITLE
WIP: Add datacite to export section

### PR DESCRIPTION
**Pull request by ines-cruz**
export2

### CODE REVIEW

File: invenio_app_rdm/config.py [hunk 1]
```diff
@@ -446,9 +446,15 @@ def files_rest_permission_factory(obj, action):
         "name": _("JSON"),
         "serializer": ("invenio_rdm_records.resources.serializers:"
                        "UIJSONSerializer")
-    }
+    },
+    "datacite": {
+       "name": _("DataCite"),
```
Comment (lnielsen) [inline invenio_app_rdm/config.py:451]:
```suggestion
       "name": _("DataCite JSON"),
```

Comment (ines-cruz) [reply invenio_app_rdm/config.py:451]: I wrote DataCite so it would be the same as in Zenodo, should i still change it to DataCite JSON?

Comment (lnielsen) [reply invenio_app_rdm/config.py:451]: It's because we'll have both a DataCite XML and DataCite JSON whereas Zenodo only has a DataCite XML.

**Comment by ppanero** [comment]
closed in favour of https://github.com/inveniosoftware/invenio-app-rdm/pull/831

**Pull request closed without merging**